packages/react-native-bpk-component-calendar/src/ios/CalendarBridge/RCTBPKCalendarManager.m

```diff
@@ -42,11 +42,11 @@ - (UIView *)view {
     return calendar;
 }
 
-RCT_EXPORT_VIEW_PROPERTY(minDate, NSDate)
-RCT_EXPORT_VIEW_PROPERTY(maxDate, NSDate)
+RCT_REMAP_VIEW_PROPERTY(minDate, rct_minDate, NSDate)
+RCT_REMAP_VIEW_PROPERTY(maxDate, rct_maxDate, NSDate)
 RCT_EXPORT_VIEW_PROPERTY(selectionType, BPKCalendarSelection)
 RCT_EXPORT_VIEW_PROPERTY(locale, NSLocale)
-RCT_EXPORT_VIEW_PROPERTY(selectedDates, NSArray<NSDate *> *)
+RCT_REMAP_VIEW_PROPERTY(selectedDates, rct_selectedDates, NSArray<NSDate *> *)
```

Comment:
Where do you find documentation for things like this? When I was investigating that calendar issue with RN 0.59, I tried looking up these macros and found nothing. Even the source code wasn't that helpful!

Reply:
I looked at the source code of React Native itself. For example here's usage in [`RCTViewManger`](https://github.com/facebook/react-native/blob/9bb5c32de9ef06942e38ea6cf5d9000c37741c37/React/Views/RCTViewManager.m#L114)